examples/visual-tests/amp-story-player/story-loaded.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+module.exports = {
+  'story loaded and player ready': async page => {
+    await page.$eval('amp-story-player', e => {
+      new Promise(resolve => {
+        e.addEventListener('ready', (readyEvent) => resolve(readyEvent));
```

Comment:
Hmm I think you have to add `return` before `new Promise` to "block" the execution of the next lines. But I'm not super familiar with Pupeteer's `$eval` (you can also ignore the `readyEvent` since it's not used).

```suggestion
      return new Promise(resolve => {
        e.addEventListener('ready', () => resolve()); 
```

Reply:
That's what I thought before too, but I tried it before and the test was just stuck there.

Reply:
Removed.